website/markdown/docs/commands/dependencies.mdx

```diff
@@ -0,0 +1,211 @@
+---
+name: Manage third-party dependencies
+excerpt: "Learn how to use Tuist's dependencies command to manage third-party dependencies."
+---
+
+import PropertiesTable from '../components/properties-table'
+import SettingsDictionaryTable from '../components/settingsdictionary-table'
+import Message from '../components/message'
+import EnumTable from '../components/enum'
+import {
+  List,
+  ListItem,
+  ListIcon,
+  ListContent,
+  ListHeader,
+  ListDescription,
+  ListList,
+} from '../components/list'
+
+# Dependencies (Chimera)
+
+## How to get started
+
+1. List requirements for third-party dependencies in `Dependencies.swift` manifest.
+2. Install dependencies using `tuist dependencies [subcommand]` command.
+3. Link dependencies with project's targets.
+4. Regenerate project.
+
+## `Dependencies.swift`
+
+Requirements for third-party dependencies are defined in `Dependencies.swift` manifest. `Dependencies.swift` need to import the framework `ProjectDescription` which contains all the classes and enums that are available for you to describe your projects.
+
+You can edit `Dependencies.swift` using `tuist edit` command.
+
+Tuist expects `Dependencies.swift` under `Tuist/` directory:
+
+```bash
+AwesomeProject
+	|- App
+	|- AppTests
+	|- Tuist
+		|- Dependencies.swift
+	|- Project.swift
+```
+
+The snippet below shows an example project manifest:
+
+```swift
+import ProjectDescription
+
+let dependencies = Dependencies([
+    .carthage(origin: .github(path: "Alamofire/Alamofire"), requirement: .exact("5.0.4"), platforms: [.macOS])
+])
+```
+
+### Dependencies
```

Comment:
I think the documentation of the models should be in [project-description.mdx](https://github.com/tuist/tuist/blob/main/website/markdown/docs/usage/project-description.mdx). As @fortmarek mentioned in a comment, this page should read more like a tutorial that guides the user through:
- How the definition of dependencies works in Tuist and the motivations behind having first-class support for it.
- Some basic examples around how to add a Carthage dependency and a package dependency.